subjects/cat.en.md

```diff
@@ -18,13 +18,15 @@ Write a program that has the same behaviour as the system's `cat` command-line.
 
 `"Alan Mathison Turing was an English mathematician, computer scientist, logician, cryptanalyst. Turing was highly influential in the development of theoretical computer science, providing a formalisation of the concepts of algorithm and computation with the Turing machine, which can be considered a model of a general-purpose computer. Turing is widely considered to be the father of theoretical computer science and artificial intelligence."`
 
+-   In case of error the program should print the error.
+
```

Comment:
May we have an example of an error in the examples?

Reply:
On it chris